openeo/api/logs.py

```diff
@@ -1,40 +1,34 @@
-class LogEntry:
-    """
-    A log entry.
-    """
-
-    def __init__(self, entry):
-        # Unique ID for the log, string, REQUIRED
-        self.id = entry['id']
-
-        # Error code, string, optional
-        self.code = entry.get("code", "")
-
-        # Severity level, string (error, warning, info or debug), REQUIRED
-        self.level = entry['level']
-
-        # Error message, string, REQUIRED
-        self.message = entry['message']
-
-        # Date and time of the error event as RFC3339 date-time, string, available since API 1.1.0
+"""
```

Comment:
this "doc block" (triple quoted string) should come after `class LogEntry` line

Reply:
Fixed.